src/library/scala/collection/mutable/StringBuilder.scala

```diff
@@ -23,7 +23,7 @@ import scala.Predef.{ // unimport char-related implicit conversions to avoid tri
   genericWrapArray => _,
   wrapCharArray => _,
   wrapString => _,
-  _
+  //_
```

Comment:
?

Reply:
Unused, but leaving it commented highlights that Predef is not root-imported.

Somewhere there's a branch for `-Yimports` to be expressed as regular imports, oh but actually that doesn't help here, I was thinking of partests.

Reply:
> leaving it commented highlights that Predef is not root-imported

only to a professional mindreader :-)

Reply:
Is there any other kind? In my defense, it's still an improvement over the warning that `_` is unused. This could be simplified to an import that only disables Predef, like `import Predef.junk`.

Reply:
Interesting idea from the feverish covid brain, that the predef import need not name an actual member to disable the root import. But actually it does error on the import.